sentry-core/src/main/java/io/sentry/core/protocol/SdkVersion.java

```diff
@@ -3,12 +3,14 @@
 import io.sentry.core.IUnknownPropertiesConsumer;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.CopyOnWriteArrayList;
 
 public class SdkVersion implements IUnknownPropertiesConsumer {
   private String name;
   private String version;
   private List<Package> packages = new CopyOnWriteArrayList<>();
+  private Map<String, String> integrations = new ConcurrentHashMap<>();
```

Comment:
@bruno-garcia is it a `Map` or should we reuse the clas `Package`? does it need to be thread-safe?

Reply:
Anything in the scope needs to be thread-safe.
integrations is a `List<String>` actually not a map, my bad.

Reply:
We're holding this off of the event not the scope so neither collection need to be synchronized